medcat/cdb.py

```diff
@@ -213,9 +213,9 @@ def add_names(self, cui: str, names: Dict, name_status: str = 'A', full_build: b
             # Name status must be one of the three
             name_status = 'A'
 
-        self.add_concept(cui=cui, names=names, ontologies=set(), name_status=name_status, type_ids=set(), description='', full_build=full_build)
+        self._add_concept(cui=cui, names=names, ontologies=set(), name_status=name_status, type_ids=set(), description='', full_build=full_build)
 
-    def add_concept(self,
+    def _add_concept(self,
```

Comment:
I think it would be useful to add the following documentation:
- How to get `names` argument using `medcat.preprocessing.cleaners.prepare_name`
- What to use for a raw name (i.e `medcat.cat.CAT.add_and_train_concept`)

Reply:
This has already been done, as we discussed on call.